jpo-ode-core/src/main/java/us/dot/its/jpo/ode/model/OdePsmMetadata.java

```diff
@@ -0,0 +1,43 @@
+package us.dot.its.jpo.ode.model;
+
+
+public class OdePsmMetadata extends OdeLogMetadata {
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 1L;
+	public enum PsmSource {
+		RSU, V2X, MMITSS, unknown
+	}
+
+	private PsmSource psmSource;
+	private String originIp;
+
+	public PsmSource getPsmSource() {
+		return psmSource;
+	}
+	public OdePsmMetadata() {
+		super();
+	}
+
+	public OdePsmMetadata(OdeMsgPayload payload) {
+		super(payload);
+	}
+
+	public OdePsmMetadata(OdeMsgPayload payload, SerialId serialId, String receivedAt) {
+
+	}
```

Comment:
Is this constructor necessary?

Reply:
No, just came over from the BSM implementation (not sure it's useful over there either).

Reply:
I'll remove it